ext/net/02_tls.js

```diff
@@ -275,6 +292,37 @@ async function startTls(
   return new TlsConn(rid, remoteAddr, localAddr);
 }
 
+const resolverSymbol = SymbolFor("unstableSniResolver");
+const serverNameSymbol = SymbolFor("unstableServerName");
+
+function createTlsKeyResolver(callback) {
+  const { 0: resolver, 1: lookup } = op_tls_cert_resolver_create();
+  (async () => {
```

Comment:
I don't like that there's a floating promise; can we store it somewhere and handle explicitly?

Reply:
This looks like a floating promise but it's actually tied to the lifetime of the resolver. Once the internal tx is shut down, the rx shuts down and this promise disappears.

Ideally we'd have streams as first-class citizens in deno_core to make this a little less awkward.